mdal/api/mdal.h

```diff
@@ -610,6 +610,36 @@ MDAL_EXPORT MDAL_DatasetH MDAL_G_addDataset(
   const int *active
 );
 
+/**
+ * Adds empty (new) 3D dataset to the group
+ * This increases dataset group count MDAL_G_datasetCount() by 1
+ *
+ * The dataset is opened in edit mode.
+ * To persist dataset, call MDAL_G_closeEditMode() on parent group
+ *
+ * Minimum and maximum dataset values are automatically calculated
+ *
+ * Only for 3D datasets
+ *
+ * \param group parent group handle
+ * \param time time for dataset (hours)
+ * \param values For scalar data, the size must be volume count
+ *               For vector data , the size must be volume count * 2 (x1, y1, x2, y2, ..., xN, yN)
+ * \param verticalLevelCount Int Array holding the numver of vertical levels for each face.
```

Comment:
numver -> number

Reply:
resolved in bc85fd85d3b09458e2d4ed04ed99644df0138a12